components/TodoList.js

```diff
@@ -58,35 +66,97 @@ function TodoList() {
         value={newTodo}
         onChange={(e) => setNewTodo(e.target.value)}
       />
+      <select value={priority} onChange={handlePriorityChange}>
+        <option value="high">High Priority</option>
+        <option value="medium">Medium Priority</option>
+        <option value="low">Low Priority</option>
+      </select>
       <button onClick={addNewTodo}>Add</button>
-      <ul>
-        {todos.map((todo) => {
-          if (editTodoId === todo.id) {
-            return (
+
+      <div style={{ display: 'flex' }}>
+        <div style={{ flex: 1 }}>
```

Comment:
You use inline style here. Better use styled-components in general.

Reply:
Thank you, it's a very good idea, I've now created a new folder for the styled components